feeding/src/util.cpp

```diff
@@ -380,12 +384,8 @@ double getDistance(
 //==============================================================================
 Eigen::Isometry3d getForqueTransform(tf::TransformListener& tfListener)
 {
-  /*
   return getRelativeTransform(
-    tfListener,
-    "/map",
-    "/j2n6s200_forque_end_effector");
-  */
+      tfListener, "/map", "/j2n6s200_forque_end_effector");
```

Comment:
Is this function being used? Why was the definition just commented out before?

Reply:
This function is only used in the (now out-dated) `bltskewerpushmain.cpp` and `bltpushingmain.cpp` demos.

I'll have another PR that cleans these unused demos out.